zerver/tornado/event_queue.py

```diff
@@ -460,19 +460,14 @@ def load_event_queues(port: int) -> None:
     global clients
     start = time.time()
 
-    # ujson chokes on bad input pretty easily.  We separate out the actual
-    # file reading from the loading so that we don't silently fail if we get
-    # bad input.
     try:
         with open(persistent_queue_filename(port), "r") as stored_queues:
-            json_data = stored_queues.read()
-        try:
-            clients = dict((qid, ClientDescriptor.from_dict(client))
-                           for (qid, client) in ujson.loads(json_data))
-        except Exception:
-            logging.exception("Tornado %d could not deserialize event queues" % (port,))
```

Comment:
I don't think removing this `except Exception` is correct; part of the intent of `except Exception` here is to ensure that a bug in the `.from_dict()` method will not prevent Tornado from starting; it's much better to drop loading old events than to fail to startup.

Reply:
The idea with `except (LookupError, TypeError, ValueError)` was to specifically catch the exceptions that could arise due to a malformed file (as opposed to say `MemoryError`). Do you still want to catch all exceptions?

Reply:
Yes, in that if the logic in `.from_dict()` can't handle the data, we want to discard the data and startup (logging/emailing the exception details), rather than crashing.  The alternative is to have an outage.

Reply:
Meh. Changed.